### PR TITLE
[8.0][product_analytic][IMP] Spanish translation

### PR DESCRIPTION
**Pull request by antespi**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Why don't you make the translation on Transifex?

**Comment by gurneyalex** [comment]
formally :+1:

File: product_analytic/i18n/es.po [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: product_analytic
+#: field:product.category,expense_analytic_account_id:0
+#: field:product.template,expense_analytic_account_id:0
+msgid "Expense Analytic Account"
+msgstr "Cuenta analítica de gastos"
```
Comment (pedrobaeza) [inline product_analytic/i18n/es.po]: s/de/para

File: product_analytic/i18n/es.po [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+#. module: product_analytic
+#: field:product.category,income_analytic_account_id:0
+#: field:product.template,income_analytic_account_id:0
+msgid "Income Analytic Account"
+msgstr "Cuenta analítica de ingresos"
```
Comment (pedrobaeza) [inline product_analytic/i18n/es.po]: s/de/para

File: product_analytic/i18n/product_analytic.pot [hunk 3]
```diff
@@ -0,0 +1,44 @@
+# Translation of Odoo Server.
```
Comment (pedrobaeza) [inline product_analytic/i18n/product_analytic.pot]: Don't add the pot file. It's useless

**Comment by antespi** [comment]
Sorry @pedrobaeza, at this point this way is more easy for me.

**Comment by antespi** [comment]
@pedrobaeza, fixes done, thanks

**Comment by pedrobaeza** [comment]
:+1: Merging...

**Comment by antespi** [comment]
@gurneyalex, @pedrobaeza Thanks all!

**Pull request merged**